kraken-std/src/kraken/std/python/tasks/pytest_task.py

```diff
@@ -7,10 +7,25 @@
 
 from kraken.common import flatten
 from kraken.core import Project, Property, TaskStatus
+from typing_extensions import Literal
 
 from .base_task import EnvironmentAwareDispatchTask
 
-# TODO (@NiklasRosenstein): Pytest coverage support
+CoverageFormat = Literal["xml", "html", "json", "lcov", "annotate", None]
+
+
+def get_coverage_path(format: CoverageFormat) -> str:
```

Comment:
A better name for this function may be `get_coverage_suffix()`? 

An enum may be a better place for the coverage formats and this method to live in.

Reply:
I first created an enum but Nick thought it was a better idea to use a Literal. I will speak with him about it.